eq-author/src/App/introduction/Design/IntroductionEditor/LegalBasisField.js

```diff
@@ -1,108 +1,60 @@
 import React from "react";
-import styled, { css } from "styled-components";
+import styled from "styled-components";
 import PropTypes from "prop-types";
 
-import { Input } from "components/Forms";
 import { colors } from "constants/theme";
-
-import iconCheck from "./icon-check.svg";
+import { Input } from "components/Forms";
 
 const LegalField = styled.div`
   display: flex;
-`;
-
-const LegalInput = styled(Input)`
-  position: absolute;
-  overflow: hidden;
-  clip: rect(0 0 0 0);
-  height: 1px;
-  width: 1px;
-  margin: -1px;
-  padding: 0;
-  border: 0;
-  &:hover,
-  &:focus {
-    border: none;
-    outline: none;
-    box-shadow: none;
-  }
+  flex-direction: column;
+  justify-content: flex-end;
 `;
 
 const LegalLabel = styled.label`
   padding: 2.5em 1.5em;
-  border-radius: 4px;
+  border-radius: 0.25em;
+  margin-bottom: 1.25em;
   border: 1px solid ${colors.bordersLight};
   flex: 1 1 33.3333333%;
-  text-align: center;
   cursor: pointer;
   display: flex;
   flex-direction: column;
-  align-items: center;
-  justify-content: center;
   background: #fff;
   box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.18);
   color: ${colors.textLight};
   position: relative;
   transition: padding 200ms ease-in-out;
 
-  &:not(:last-of-type) {
-    margin-right: 1em;
-  }
-
   &:focus-within {
     border-color: ${colors.blue};
     outline-color: ${colors.blue};
     box-shadow: 0 0 0 3px ${colors.tertiary};
   }
-
-  &::before {
-    content: url(${iconCheck});
-    display: inline-block;
-    width: 1em;
-    height: 1em;
-    margin: 0 auto;
-    z-index: 1;
-    transform: scale(0);
-    opacity: 0;
-    transition: all 100ms ease-out 100ms;
-    margin-top: -1.5em;
-    margin-bottom: 0.5em;
-  }
-
-  ${props =>
-    props.selected &&
-    css`
-      border: 1px solid ${colors.primary};
-      padding: 3em 1.5em 2em;
-      &::before {
-        transform: scale(1);
-        opacity: 1;
-      }
-    `};
 `;
 
 const LegalTitle = styled.span`
-  font-size: 0.85em;
   font-weight: bold;
   letter-spacing: 0;
-  margin: 0 0 1em;
+  margin-left: 3em;
   color: ${colors.text};
-`;
-
-const LegalNotice = styled.span`
-  font-weight: bold;
-  margin-bottom: 1em;
-  width: 8em;
+  display: flex;
+  margin-top: -1em;
```

Comment:
I'm unsure on if I'm a fan of negative margins. It suggests, to me at least, that the component should be closer to something else. In my opinion, it would feel tidier if you can instead reduce the bottom spacing on the component that is causing the too-big of a gap.